bedrock/firefox/views.py

```diff
@@ -542,8 +542,11 @@ def new(request):
                 template = 'firefox/new/batm/scene2.html'
             else:
                 template = 'firefox/new/onboarding/scene2.html'
-        elif lang_file_is_active('firefox/new/onboarding', locale):
-            template = 'firefox/new/onboarding/scene2.html'
+        elif lang_file_is_active('firefox/new/onboarding' 'firefox/new/batm', locale):
+            if experience == 'batmprivate':
+                template = 'firefox/new/batm/scene2.html'
+            else:
+                template = 'firefox/new/onboarding/scene2.html'
```

Comment:
I'm not sure this logic is sound. If `experience` is *not* `batmprivate`, then we don't care if the `firefox/new/batm` lang file is active, correct?

Should this instead be two `elif`s?

```
# aside from en-US, this will only be seen for 'de' locale
elif lang_file_is_active('firefox/new/batm', locale) and experience == 'batmprivate':
    template = `firefox/new/batm/scene2.html`
elif lang_file_is_active('firefox/new/onboarding', locale):
    template = `firefox/new/onboarding/scene2.html`
````

I had a little trouble following the localization conversation, but it sounded to me like this page will only be seen for `de` and `en-US`, which is why I added the comment in the code snippet above. Disregard if I've misunderstood the L10n stuff.